piqe_ocp_lib/tests/resources/test_ocp_resource_quotas.py

```diff
@@ -0,0 +1,162 @@
+import logging
+import pytest
+import random
+from piqe_ocp_lib.api.resources import OcpResourceQuota
+from piqe_ocp_lib import __loggername__
+
+logger = logging.getLogger(__loggername__)
+
+five_digit_number = ''.join(random.sample('0123456789', 5))
+NAMESPACE = "default"
```

Comment:
Might be a nitpick, but we might want to consider always testing on Projects/Namespaces that we create.

Reply:
@btjd  valid point. How should we approach for consistency? Should we add a fixture in the conftest that creates a test namespace (separate from the one the test_ocp_project uses) that all other test could use. Or do we want to handle it at the test module level and every test module should create their own namespace to test in?

Reply:
I think we should handle it at the test module level. I can see some examples where the test requires creating more than one project that gets used by the test. The first example that comes to mind is installing an operator in more than one target namespace. Feel free to check out the test_ocp_operators test for more details.